src/status_im/contexts/wallet/events.cljs

```diff
@@ -375,3 +375,20 @@
             {:title   title
              :subject title
              :message content})]]}))
+
+(rf/reg-event-fx :wallet/secret-phrase
+ (fn [{:keys [db]} [{:keys [secret-phrase random-phrase]}]]
+   {:db (-> db
+            (assoc-in [:wallet :ui :secret-phrase] secret-phrase)
```

Comment:
coming back to the data model here this should be nested in a category of it's own imo.
i.e
`wallet :ui :create-account :secret-phrase` or something to that effect.
cc @smohamedjavid, @briansztamfater @ulisesmac

Reply:
Agreed. Also, the name of the event (`:wallet/secret-phrase`) doesn't look descriptive enough to me.

Reply:
@briansztamfater the event `:wallet/secret-phrase` is only dispatching the secret phrase into app db. Any suggestions?

Reply:
Maybe. something like `:wallet/store-secret-phrase` or `:wallet/set-secret-phrase`? WDYT?